jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: classic-docs
+title: "API v2 Introduction"
+short-title: "Introduction"
+description: "introduction to the CircleCI API v2"
+categories: [getting-started]
+order: 1
+---
+
+The CircleCI API v2 is a new API that may be used by internal and external users to make API calls to retrieve detailed information about workflows and pipelines.
+
+## Introduction to API v2
+
+CircleCI API v2 enables you to use a brand-new API with a set of endpoints and several new features that improve the API experience, in addition to optimizing how you use the API for your jobs. Some of these features include the ability to:
```

Comment:
Might need to tweak this language to represent the "mixed" stability of the API. Specifically, the current categories of API v2 Endpoints are:
* Authentication
* User (Preview)
* Pipeline (Preview)
* Project (Preview)
* Job (Preview)
* Workflows (Preview)

In the near future it will just be Authentication, Pipeline and Workflow that will be marked Stable, while Job, User, Project will remain in Preview.

Reply:
fixed